cmd/root.go

```diff
@@ -11,13 +11,9 @@ import (
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "opcap",
-	Short: "A brief description of your application",
-	Long: `A longer description that spans multiple lines and likely contains
-examples and usage of using your application. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "A CLI that assesses the capability levels of operators from a specified catalog source",
+	Long: `The OpenShift Operator Capabilities Tool (opcap) is a command line interface that assesses
+the capability levels of operators from a specified catalog source`,
```

Comment:
This Long description looks pretty much like the short. Would be nice to put at least one example showing how to use the tool.

Reply:
Ok. Doesn't look like a long description though. But it's not a blocker.